backend/src/main/java/com/example/backend/controller/LessonController.java

```diff
@@ -23,4 +24,9 @@ public Lesson saveNewLesson(@RequestBody LessonDTO lessonDTO) {
     public List<Lesson> getAllLessons(){
         return lessonService.getAllLessons();
     }
+
+    @GetMapping("/student/{studentId}")
```

Comment:
if you don't have many collections with many Ids, this would be more readable: 
/student/{id}